vector/src/main/res/values/strings.xml

```diff
@@ -2566,8 +2572,20 @@
     <string name="auth_pin_reset_title">Reset pin</string>
     <string name="auth_pin_new_pin_action">New pin</string>
     <string name="auth_pin_reset_content">To reset your PIN, you\'ll need to re-login and create a new one.</string>
+    <string name="settings_security_application_protection_title">Protect access to the application.</string>
+    <string name="settings_security_application_protection_summary">Protect access to the application using pin code and biometrics.</string>
+    <string name="settings_security_application_protection_screen_title">Configure protection</string>
     <string name="settings_security_pin_code_title">Enable PIN</string>
     <string name="settings_security_pin_code_summary">If you want to reset your PIN, tap Forgot PIN to logout and reset.</string>
+    <string name="settings_security_pin_code_use_biometrics_title">Use biometrics</string>
+    <string name="settings_security_pin_code_use_biometrics_summary_on">Depending on your device, you will be able to unlock using fingerprints, face recognition, iris recognition, etc.</string>
+    <string name="settings_security_pin_code_use_biometrics_summary_off">PIN code is the only way to unlock the application.</string>
+    <string name="settings_security_pin_code_notifications_title">Display complete notifications</string>
+    <string name="settings_security_pin_code_notifications_summary_on">Show full notification details, including rooms details and message contents.</string>
+    <string name="settings_security_pin_code_notifications_summary_off">Only display number of unread messages in a simple notification.</string>
+    <string name="settings_security_pin_code_grace_period_title">Enable grace period</string>
```

Comment:
I don't know if this is a well known phrase but I'm not super confident in the following suggestions:

Require PIN after 2 minutes

Reply:
I replace "Enable grace period" with "Require PIN after 2 minutes"